packages/cli/generators/relation/index.js

```diff
@@ -164,11 +171,18 @@ module.exports = class RelationGenerator extends ArtifactGenerator {
         name: parameter,
         message: message,
         choices: modelList,
-        when: this.artifactInfo[parameter] === undefined,
+        when: !this.artifactInfo[parameter],
```

Comment:
Interesting. IIUC, this change is required because `this.artifactInfo[parameter]` can be set to `null`, `false` or `0`? Is that expected & intentional?

Reply:
No matter. Before prompt, `this.artifactInfo[parameter]` is undefined or equal to `this.options[parameter]` value from the `modelList`. `modelList` can't be empty.